sematic/db/migrations/20230609124213_pre_alembic_parity.sql

```diff
@@ -0,0 +1,47 @@
+-- migrate:up
+
+UPDATE runs SET function_path = '' WHERE function_path IS NULL;
```

Comment:
for unclear reasons, we all currently do have runs where `function_path` is null.  this should not happen, and we want to fail hard when this occurs in the future.  all past instances of this in `sash`, `josh`, and `tudor`s databases are from around a month ago, clustered around very specific dates.  the suspicion is that we had a release that contained an unknown bug that caused this for a brief period of time.

for the past runs where this is null, those runs are broken anyway, so we're just setting this to a non-null value so that postgres can start enforcing the non null constraint on the column going forward.

Reply:
Rather than the empty string, can we set it to "UNKNOWN.UNKNOWN"? I can see the empty string causing weirdness in the UI.